test/CheckoutSdkTest/Instruments/Four/BankAccountFieldFormattingTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Checkout.Instruments.Four.Get;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Checkout.Instruments.Four
+{
+    public class BankAccountFieldFormattingTest : SandboxTestFixture
+    {
+        public BankAccountFieldFormattingTest() : base(PlatformType.FourOAuth)
+        {
+        }
+
+        [Fact]
+        private async Task ShouldGetBankAccountFieldFormatting()
+        {
+            BankAccountFieldQuery query = new BankAccountFieldQuery()
+            {
+                AccountHolderType = Common.Four.AccountHolderType.Individual,
+                PaymentNetwork = PaymentNetwork.Local
+            };
+
+            var response = await FourApi.InstrumentsClient().GetBankAccountFieldFormatting(Common.CountryCode.GB, Common.Currency.GBP, query);
+
+            Assert.NotNull(response);
```

Comment:
Please include the same assertions that exist in Java `BankAccountFieldFormattingTestIT`

Reply:
Also, we are using Shouldly library for assertions, so you shold implement something like:
`response.ShouldNotBeNull();`

Please use the same patter in all your tests,